db/schema.rb

```diff
@@ -79,7 +79,7 @@
   end
 
   create_table "asp_users", force: :cascade do |t|
-    t.string "email", default: "", null: false
```

Comment:
Ah on se demandait avec Dylan d'où ca venait ça? C'est mal passé dans une migration?

Reply:
excellente question à laquelle je n'ai pas la réponse, hors de question de mettre un default par contre non ?

Reply:
bah ouais

Reply:
@ac-clermont-dylan On a notre réponse, c'est une mig ou une update qui a pas du être checked in.